nextcloud_update.sh

```diff
@@ -309,7 +309,22 @@ fi
 # Update all Nextcloud apps
 if [ "${CURRENTVERSION%%.*}" -ge "15" ]
 then
-    occ_command app:update --all
+    occ_command app:update --all >> Updated_Apps
```

Comment:
One `>` would be enough here as it only is valid one time.

Reply:
i changed it so not anymore relevant